src/lang/af/seat.php

```diff
@@ -212,8 +212,11 @@
     'add_affiliations'            => 'Voeg Affiliasie By',
     'current_affiliations'        => 'Huidige Affiliasies',
     'available_users'             => 'Beskikbare Gebruikser',
+    'available_groups'            => 'Available Groups',
```

Comment:
In addition, in case you don't have a translation, just not provide it. Default will be `en`